java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1010,6 +1011,82 @@ public void lrem_existing_non_existing_key_and_type_error(BaseClient client) {
         assertEquals(0, client.lrem("non_existing_key", 0, "value").get());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void lpos_with_options(BaseClient client) {
```

Comment:
Looks like we test lpos with and without options here
```suggestion
    public void lpos(BaseClient client) {
```